src/styles/blocks/card.scss

```diff
@@ -0,0 +1,97 @@
+.card {
+  width: $card-width;
+  padding: 32px 16px 16px;
+  margin-right: $card-hor-mar;
+  margin-bottom: $card-bot-mar;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &:nth-last-child(-n + 4) {
+    margin-bottom: 46px;
+  }
```

Comment:
These margins are related to the catalog block, not to card. Move this styles from here

Reply:
fixed